examples/mock-awx/mock-awx.go

```diff
@@ -0,0 +1,38 @@
+package main
```

Comment:
Add the copyright header, like in all the other files of the project.

Reply:
:+1: done